.github/workflows/release.yml

```diff
@@ -0,0 +1,419 @@
+name: release
+
+on:
+  push:
+    tags:
+      - 'v*'
+
+permissions:
+  contents: write
```

Comment:
I think only the create-release job needs write permissions, so it's better to set this to read, and set the write permission only in the create-release job.

Reply:
Done in https://github.com/FirebirdSQL/firebird-odbc-driver/pull/245/commits/37b83fb0a64fe750774f45610720ef2ea00b4252